kitsune/sumo/jinja2/base.html

```diff
@@ -199,9 +199,23 @@
   </div>
 </header><!--/.mzp-c-navigation -->
 
+{% block hidden_search_masthead %}
+<!-- This is only displayed when the instant search is active.
+     This block is overriden in home.html and product.html,
+     and can likely be removed when frontend search is replaced. -->
+<section class="home-search-section sumo-page-section shade-bg hidden-search-masthead">
```

Comment:
@kkellydesign I looked into the test that you asked about.

```
======================================================================
FAIL: kitsune/questions/tests/test_templates.py:QuestionsTemplateTestCase.test_product_filter
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/kitsune/questions/tests/test_templates.py", line 1133, in test_product_filter
    check('all', [q1, q2, q3])
  File "/app/kitsune/questions/tests/test_templates.py", line 1127, in check
    eq_(len(expected), len(doc('.questions > section')))
AssertionError: 3 != 4
```
It is finding 4 instead of 3 because the `<body>` element has a class of `questions`, so this new `<section>` element is being found.

You will either need to futz with class names in the templates or the selector in the test (`'.questions > section'`).

I hope that helps. Let me know if you need anything else.